File: poo/cplus2/struct_sec/angulos.cc
#include <iostream>

using namespace std;

class Angulo {
private:
    double radianes;

public:
    void leerAngulo() {
        cout << "Ingrese el angulo en radianes: ";
        cin >> radianes;
    }

    void printAngulo() {
        cout << "El angulo introducido es: " << radianes << " radianes" << endl;
    }
};

int main() {
    Angulo ang;
    ang.leerAngulo();
    ang.printAngulo();
    return 0;
}

File: poo/java/struct_sec/Angulos.java
import java.util.Scanner;

public class Angulos {
    private double radianes;

    public void leerAngulo(Scanner leer) {
        System.out.print("Ingrese el angulo en radianes: ");
        radianes = leer.nextDouble();
    }

    public void printAngulo() {
        System.out.println("El angulo introducido es: " + this.radianes + " radianes");
    }
    public static void main(String[] args) {
        Scanner leer = new Scanner(System.in);
            
        Angulos ang = new Angulos();
        ang.leerAngulo(leer);
        ang.printAngulo();
            
    }
}
